deploy.sh

```diff
@@ -0,0 +1,36 @@
+if ([ $TRAVIS_PULL_REQUEST == "false" ])
+then
+    #Clone the latest gh-pages
+    git clone https://github.com/ScottLogic/StockFlux.git --branch gh-pages gh-pages
+
+    #Get line with version from the file -> get the second word -> remove brackets
+    VERSION=$(grep "version" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    #Get line with the release type (develop/master) from the file -> get the second word -> remove brackets
+    TYPE=$(grep "type" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    rm -rf ./gh-pages/$TYPE
+    cp -r ./public ./gh-pages/$TYPE
+    rm -rf ./gh-pages/$VERSION
```

Comment:
I can see the advantage for fixing up issues, but I'm a bit wary on the versions overwrite, especially if we changed the Travis config to build on other branches etc., but I think fine for this PR.  The planned structure looks good and we can tweak the actually deploy.sh as needed later